dotnet/src/Planners/Planners.Core/Utils/EmbeddedResource.cs

```diff
@@ -4,7 +4,7 @@
 using System.Reflection;
 using Microsoft.SemanticKernel.Diagnostics;
 
-namespace Microsoft.SemanticKernel.Planners.Action;
+namespace Microsoft.SemanticKernel.Planners;
```

Comment:
Namespace name doesn't match folder structure. You can:
- Make it `Microsoft.SemanticKernel.Planners.Utils`.
- Remove namespace so it will use root namespace by default.